ghdocs/TUTORIAL.md

```diff
@@ -323,27 +507,238 @@ To accomplish this, let's add the following new styles to app.css:
 
 These styles are going to override the default component styling and will help our TextField and Button stand out with some minor positioning adjustments.
 
-Now let's tweak our list a bit. For our purposes, we really want to just have a title and description with the x and checkmark. Since we don't need the paper clip and flag, let's modify one list item to look like the following:
+**Hang on, this part gets a bit tricky**
+
+But now we'd really like to tweak our component by adding a title and a description along with icons for an x and a checkmark. Since we don't need the paper clip and flag, go ahead and remove all `<div class="ms-ListItem is-selectable"></div>` elements **except the first one** so we have one element to work with.
+
+Inside of the first `.ms-ListItem` remove the elements` <div class="ms-ListItem-action"><i class="ms-Icon ms-Icon--mail"></i></div>` and `<div class="ms-ListItem-action"><i class="ms-Icon ms-Icon--trash"></i></div>`. Then change the icons to the x and the check: replace `ms-Icon ms-Icon--flag` with `ms-Icon ms-Icon--x` and `ms-Icon ms-Icon--pinLeft` with `ms-Icon ms-Icon--check`. Our new list HTML should look like the following:
 
```

Comment:
Probably want to mention that you remove the "Meeting Notes" part and you're also including the "is-unread" class.